Cargo.toml

```diff
@@ -1,22 +1,23 @@
 [package]
 name = "passage_flex"
-version = "0.1.1"
+version = "0.1.2"
 edition = "2021"
 authors = ["support@passage.id"]
 description = "Provides verification of server-side authentication for applications using Passage Passkey Flex"
 homepage = "https://github.com/passageidentity/passage-flex-rust"
 repository = "https://github.com/passageidentity/passage-flex-rust"
 readme = "README.md"
 license = "MIT"
+include = ["/src", "README.md"]
 
 [dependencies]
```

Comment:
On the changes below:

> Leaving off the caret is a simplified equivalent syntax to using caret requirements. While caret requirements are the default, it is recommended to use the simplified syntax when possible.
>
> log = "^1.2.3" is exactly equivalent to log = "1.2.3".

([docs](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#caret-requirements))